app/services/data_migrations/backfill_export_type.rb

```diff
@@ -4,28 +4,57 @@ class BackfillExportType
     MANUAL_RUN = false
 
     def change
-      data_exports = DataExport.all.where(export_type: nil)
+      ActiveRecord::Base.connection.execute('SET statement_timeout = 0')
```

Comment:
Why do we need this? The query doesn't look too heavy

Reply:
Prompted by the root cause, but with the other changes we've made here (not retrieving the data column in particular) I don't think is needed. Will remove.